### PR TITLE
unpin mpl

### PR DESCRIPTION
**Pull request by richrines1**

### CODE REVIEW

File: qiskit-superstaq/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 general-superstaq~=0.5.6
-matplotlib~=3.0
```
Comment (saeubank) [inline qiskit-superstaq/requirements.txt:2]: Why do we even have matplotlib in qiskit-superstaq? Do/should we be making graphs?

File: supermarq-benchmarks/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
 cirq-superstaq~=0.5.6
-matplotlib<=3.5.2
```
Comment (saeubank) [inline supermarq-benchmarks/requirements.txt:2]: I think we need this in supermarq-benchmarks.

Comment (saeubank) [reply supermarq-benchmarks/requirements.txt:2]: Used in https://github.com/Infleqtion/client-superstaq/blob/6f2ffe50658096b97062f5d9c3a5deff30938c77/supermarq-benchmarks/supermarq/plotting.py#L255

Comment (richrines1) [reply supermarq-benchmarks/requirements.txt:2]: it's already a requirement of cirq-core - all this was doing was limiting the max version (for reasons i can't really remember but which no longer seem relevant?)

File: qiskit-superstaq/requirements.txt (hunk 1 above)
Comment (richrines1) [reply qiskit-superstaq/requirements.txt:2]: good question. i'll try removing it

**Review by saeubank — APPROVED** [review]

**Pull request merged**